projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -293,6 +292,7 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
         }
         if (super.selection.length) {
             this.selectionService.clear(this.id);
+            this.filterValue = this.searchValue = typeof event === 'string' ? event : event.target.value;
```

Comment:
I left this line so that as there is a set value in the combo and we try to filter the combo by writing in the input, it should be filtered.